bin/i18n/resources/apps/musiclab_libraries.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../../i18n_script_utils'
+
+module I18n
+  module Resources
+    module Apps
+      module MusiclabLibraries
+        DIR_NAME = 'musiclab_libraries'
+        # Allows temporarily sourcing input strings from a new version of our Music Lab
+        # launch library (music-library-launch2024-v2), which will eventually
+        # replace music-library-launch2024.
+        # Once we've overwritten the contents of music-library-launch2024 with
+        # what's in music-library-launch2024-v2, we can remove this
+        # and replace it with a simple array of library filenames
+        # (ie, %w[music-library-intro2024 music-library-launch2024-v2])
```

Comment:
I think it might be worth keeping the mapping system in place, since it's quite possible we'll have future library updates that we want to pre-translate like this.

Reply:
Cool -- tangentially, do you have any concern about the stability of the v2/other in development files? I guess my concern about using this approach longer term is whether we might end up with partial/bad content in the translation pipeline as we're doing active development on a new library.

Reply:
We're fairly careful about this file, testing it regularly as we add content to it.  Certainly we might tweak some strings before release, but I think we can deal with those cases as they arise, similar to how existing loc'ed strings sometimes get changed.